src/routes/ap.rs

```diff
@@ -1,51 +1,22 @@
 use itertools::Itertools;
-use rocket::{Outcome, Route};
+use rocket::Route;
 use rocket::response::Content;
-use rocket::request::{self, FromRequest, Request};
-use rocket::http::{Accept, ContentType, MediaType};
+use rocket::http::ContentType;
 use rocket_contrib::Json;
 
 use db;
-use db::models::Account;
+use db::models::{Account, Status};
 use error::Perhaps;
-use activitypub::{ActivityStreams, AsActivityPub};
+use activitypub::{ActivityGuard, ActivityStreams, AsActivityPub};
```

Comment:
just curious about the naming here: can `ActivityStreams` represent multiple streams in a single object as well? it looks singular in the returns.

Reply:
no, it can't! thanks for pointing this out. it means more 'this has the _content type_ of ActivityStreams' and not 'this _is_ an activitystream', thus the name. that's potentially ambiguous and i should maybe change it...